content/v2/domains/name-servers.markdown

```diff
@@ -74,7 +74,21 @@ Name | Type | Description
 ##### Example
 
 ~~~json
-{}
+"{
+  "name_servers": [
+    "ns1.example.com",
+    "ns2.example.com"
+  ]
+}"
```

Comment:
@jodosha I think we should just support one way, unless it proves to be confusing. I'd only document (and support) the array version.